notebooks/234-blip-visual-language-processing/234-blip-visual-language-processing.ipynb

```diff
@@ -0,0 +1,766 @@
+{
```

Comment:
<p>Add a flowchart for better explanation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>the same comment about text encoder</p>

Reply:
maybe as follow up in next request